exonum-java-binding/core/rust/build.rs

```diff
@@ -8,4 +11,12 @@ fn main() {
     if cfg!(target_os = "macos") && var_os("ROCKSDB_STATIC").is_some() {
         println!("cargo:rustc-link-lib=dylib=c++");
     }
+
+    //let exonum_protos = get_exonum_protobuf_files_path();
```

Comment:
Why is the code commented?

Reply:
It's left for future use. If not only custom protos will be required.

Reply:
Please remove, as there is no need in this code right now.